tests/FakeItEasy.Specs.netstd/ICloneable.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace FakeItEasy.Specs
+{
+    public interface ICloneable

```

Comment:
This type doesn't exist in .NET Standard. The `Clone` method isn't actually necessary, but I included it for consistency with the BCL type.